.github/workflows/cloudflare.yaml

```diff
@@ -0,0 +1,34 @@
+name: Cloudflare Conformance Tests
+on:
+  schedule:
+    - cron: "0 0 * * *"
+  push:
+    tags: ["v*"]
+  pull_request:
+    branches: [main]
+  workflow_dispatch:
+jobs:
+  test:
+    if: github.event_name != 'pull_request' || startsWith(github.event.pull_request.title, 'Release ')
```

Comment:
This is great, but we need at least minimal documentation.

packages/connect-conformance/README.md has plenty of space. Can you explain there that Cloudflare conformance tests run daily, and for every release PR, and all other conformance tests run for every commit?

Then we can update .github/RELEASING.md step 4, and clarify "The PR title must start with `Release ` to run conformance tests." (linking to the conformance readme).